contents/handbook/people/team-structure/team-structure.md

```diff
@@ -48,32 +48,45 @@ We've organised the team into small teams that are multi-disciplinary. [You can
 - [Michael Matloka](/handbook/company/team#michael-matloka-software-engineer) (Full Stack Engineer)
 - [Paul D'Ambra](/handbook/company/team#paul-dambra-software-engineer) (Full Stack Engineer)
 - _[Lottie Coxon](/handbook/company/team#lottie-coxon) (Graphic Designer) - pairs with Chris Clark_
-
-#### [App West](app-west)
-- [Eric Duong](/handbook/company/team#eric-duong-software-engineer) (Team lead, Full Stack Engineer)
-- [Neil Kakkar](/handbook/company/team#neil-kakkar-software-engineer) (Full Stack Engineer)
-- [Li Yi Yu](/handbook/company/team#li-yi-yu-full-stack-engineer) (Full Stack Engineer)
-- _[Chris Clark](/handbook/company/team#chris-clark) (Product Designer)_ - _Works across App West and App East_
+- _Hiring for a [full stack engineer](careers/full-stack-engineer)_
+- _Hiring for a [full stack engineer](careers/full-stack-engineer)_
+- Product: Supported by Annika Schmid
 
 #### [Session Recordings](session-recordings)
-- [Ben White](/handbook/company/team#ben-white-full-stack-engineer) (Full Stack Engineer)
+- [Ben White](/handbook/company/team#ben-white-full-stack-engineer) (Team lead, Full Stack Engineer)
 - [Alex Kim](/handbook/company/team#alex-kim-full-stack-engineer) (Full Stack Engineer)
-- [Rick Marron](/handbook/company/team#rick-marron-full-stack-engineer) (Full Stack Engineer)
+- [Emanuele Capparelli](/handbook/company/team#ben-white-full-stack-engineer) (Growth Engineer)
+- _Hiring for a [full stack engineer](careers/full-stack-engineer)_
+- Product: Supported by Annika Schmid
+
+#### [App West - Feature flags / experiments / cohorts / SQL](app-west)
+- [Eric Duong](/handbook/company/team#eric-duong-software-engineer) (Team lead, Full Stack Engineer)
+- [Li Yi Yu](/handbook/company/team#li-yi-yu-full-stack-engineer) (Full Stack Engineer)
+- [Neil Kakkar](/handbook/company/team#neil-kakkar-software-engineer) (Full Stack Engineer)
+- _[Chris Clark](/handbook/company/team#chris-clark) (Product Designer)_ - _Works across App West and Product Analytics_
+- _Hiring for a [full stack engineer](careers/full-stack-engineer)_
+- Product: Supported by Annika Schmid
 
 #### [Platform](platform)
 - [James Greenhill](/handbook/company/team#james-greenhill-software-engineer) (Team lead, Data/Infra Engineer)
-- [Karl-Aksel Puulmann](/handbook/company/team#karlaksel-puulmann-software-engineer) (Full Stack Engineer)
-- [Tiina Turban](/handbook/company/team#tiina-turban-software-engineer) (Full Stack Engineer)
-- [Yakko Majuri](/handbook/company/team#yakko-majuri-software-engineer) (Full Stack Engineer)
-- [Marcus Hyett](/handbook/company/team#marcus-hyett) (Product Manager)
+- Ellie Huxtable
 - [Guido Iaquinti](/handbook/company/team#guido-iaquinti-software-engineer) (Site Reliability Engineer)
+- _Hiring for an [Site Reliability Engineer](site-reliability-engineer-kubernetes)_
+- Product: Supported by Luke Harries
+
+#### [Pipeline](pipeline)
```

Comment:
I wonder why not the existing Ingestion name?

Reply:
If we are moving towards the team owning the CDP platform then they own ingestion as well as exports to the various apps. Ingestion didn't seem to encompass this to me. I'll discuss with their team further based on which OKRs they end up deciding on with a slight preference towards Ingestion as there's no need to rename just for the sake of it